packages/next/server/web/spec-extension/request.ts

```diff
@@ -31,6 +31,30 @@ export class NextRequest extends Request {
     }
   }
 
+  [Symbol.for('edge-runtime.inspect.custom')]() {
+    return {
+      cookies: this.cookies,
+      geo: this.geo,
+      ip: this.ip,
+      nextUrl: this.nextUrl,
+      url: this.url,
+      // rest of props come from Request
```

Comment:
Ideally, we don't have to redeclare all these props. Something weird is happening, I have to investigate